### DISCUSSION TITLE
AcmeBot Support IIS Server & Azure App Proxy SSL Cert

### DISCUSSION THREAD

**Discussion by pir8g33k** [polymind-inc/acmebot#586] [Q&A] [answered]
Do you have plan to support IIS server & Azure App Proxy for ssl cert renewal with letsencrypt?

**Comment by shibayan** [comment, marked as answer]
Please throw feedback so that IIS and App Proxy can support Key Vault certificates. This app is loosely coupled and will not be involved beyond the Key Vault.

**Comment by wonsil** [comment]
I use Azure Arc with the Key Vault extension to install the certificates to IIS from Key Vault.

https://github.com/epiusers-help/AutoDevCerts/blob/main/AzureArc.md
